src/PresentationalComponents/RulesTable/RulesTable.spec.ct.js

```diff
@@ -0,0 +1,642 @@
+      v.values.forEach((filterValues) => {
+        it(`${k}: ${filterValues}`, () => {
+          // disabled recommendations have Disabled in their names
+          //had to add [2] index to point to the row values
+          let modifiedData = _.cloneDeep(data[2]);
+          modifiedData.forEach((it) => {
+            if (it.disabled) {
+              it.description = it.description + ' \nDisabled';
+            }
+          });
+          const filters = { [k]: filterValues };
+          checkFiltering(
+            filters,
+            filtersConf,
+            _.map(filterData(filters, modifiedData), 'description').slice(
+              0,
+              DEFAULT_ROW_COUNT
+            ),
+            'Name',
+            TABLE_HEADERS,
+            'No matching recommendations found',
+            true,
+            true
+          );
+        });
+      });
+    });
+  }); */
+
+  // DOESN"T WORK NEED A REWRITE
+  /* describe('combined filters', () => {
+    beforeEach(() => {
+      cy.intercept('*', {
+        statusCode: 201,
+        body: {
+          ...fixtures,
+        },
+      }).as('call');
+
+      const store = getStore();
+
+      mount(
+        <MemoryRouter>
+          <IntlProvider
+            locale={navigator.language.slice(0, 2)}
+            messages={messages}
+          >
+            <Provider store={store}>
+              <RulesTable />
+            </Provider>
+          </IntlProvider>
+        </MemoryRouter>
+      );
+    });
+
+    filterCombos.forEach((filters) => {
+      it(`${Object.keys(filters)}`, () => {
+        // disabled recommendations have Disabled in their names
+        let modifiedData = data[2];
+        modifiedData.forEach((it) => {
+          if (it.disabled) {
+            it.description = it.description + ' \nDisabled';
+          }
+        });
+        checkFiltering(
+          filters,
+          filtersConf,
+          _.map(filterData(filters, modifiedData), 'description').slice(
+            0,
+            DEFAULT_ROW_COUNT
+          ),
+          'Name',
+          TABLE_HEADERS,
+          'No matching recommendations found',
+          true,
+          true
+        );
+      });
+    });
+  });
+
+  it('clears text input after Name filter chip removal', () => {
+    filterApply({ name: 'cc' });
+    // remove the chip
+    cy.contains(CHIP_GROUP, 'Name')
+      .find('button')
+      .click()
+      .then(() => {
+        expect(window.location.search).to.not.contain('text=');
+      });
+    cy.get(TOOLBAR_FILTER).find('.pf-c-form-control').should('be.empty');
+  });
+
+  it('clears text input after resetting all filters', () => {
+    filterApply({ name: 'cc' });
+    // reset all filters
+    cy.get(TOOLBAR)
+      .find('button')
+      .contains('Reset filters')
+      .click()
+      .then(() => {
+        expect(window.location.search).to.not.contain('text=');
+      });
+    cy.get(TOOLBAR_FILTER).find('.pf-c-form-control').should('be.empty');
+  }); */
+});
```

Comment:
these two passed to me => uncomment?

Reply:
Are you sure? 
![image](https://user-images.githubusercontent.com/62722417/194817367-f1d8e1e7-2688-48cd-832c-22821ec300bb.png)

Reply:
yes, it did worked for me. but whatever, let's check it later and not be blocked by this now